### DISCUSSION TITLE
save vs save_meshio

### DISCUSSION THREAD

**Discussion by MPenaR** [pyvista/pyvista#4964] [Q&A]
Is there any difference between `object.save` and `pyvista.save_meshio(object)`?? I couldn't infer it from the documentation

**Comment by banesullivan** [comment]
`save_meshio` uses the [`meshio` library](https://github.com/nschloe/meshio) to perform the file IO whereas `save` uses writers from VTK. If saving to a VTK format, definitely use the `save` method.

**Comment by banesullivan** [comment]
Some more details can be found at: https://docs.pyvista.org/getting-started/simple.html#exporting

**Comment by MPenaR** [comment]
Thank you for crearing it up
